components/Main/ClientChatPage.tsx

```
@@ -256,8 +304,25 @@ export default function ClientChatPage({
                 key={`ai-${nextMessage.id || `${index}-response`}`}
                 className="mt-4 text-gray-800 dark:text-gray-200 group relative"
               >
-                <div className="prose prose-sm dark:prose-invert max-w-none break-words">
+                <div className="prose prose-sm dark:prose-invert max-w-none break-words prose-spacing">
                   <ReactMarkdown
+                    components={{
+                      h1: ({ node, ...props }) => (
```

ESLint

[error] 'node' is defined but never used.

(@typescript-eslint/no-unused-vars)